support/Python/__main__.py

```diff
@@ -31,6 +31,7 @@ def list_commands(self, ctx):
             "interpolate-to-mesh",
             "interpolate-to-points",
             "plot",
+            "render-bbh",
```

Comment:
move this into the `Render3D` subdirectory please and add to the subcommand there, which already has `domain` and `slice` subcommands.